mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -36,8 +36,10 @@ import (
 )
 
 const (
-	//taken from android-wireguard project
+	// Taken from android-wireguard project
 	androidTunMtu = 1280
```

Comment:
On wireguard-go default MTU is 1420, but on wireguard-android it is 1280. Need to check why it is lower, do you have any ideas? Just want to be sure it will not impact performance.

Reply:
Most likely it due to IPv6 compatibility. Minimum IPv6 packet MTU should be 1280. Why it is that low? Not sure, most likely due to bet possible traversal without actual fragmentation.